core/editor/src/input/input_mapper.rs

```diff
@@ -235,6 +235,10 @@ impl Default for Mapping {
 			entry! {action=DocumentMessage::NudgeSelectedLayers(NUDGE_AMOUNT, -NUDGE_AMOUNT), key_down=KeyArrowRight, modifiers=[KeyArrowUp]},
 			entry! {action=DocumentMessage::NudgeSelectedLayers(NUDGE_AMOUNT, NUDGE_AMOUNT), key_down=KeyArrowRight, modifiers=[KeyArrowDown]},
 			entry! {action=DocumentMessage::NudgeSelectedLayers(NUDGE_AMOUNT, 0.), key_down=KeyArrowRight},
+			entry! {action=DocumentMessage::MoveSelectedLayer(-1), key_down=KeyLeftBracket, modifiers=[KeyControl]},
+			entry! {action=DocumentMessage::MoveSelectedLayer(1), key_down=KeyRightBracket, modifiers=[KeyControl]},
+			entry! {action=DocumentMessage::MoveSelectedLayer(-999999), key_down=KeyLeftBracket, modifiers=[KeyControl, KeyShift]},
```

Comment:
These key binds (ctrl+shift+[ & ]) aren't working, I'm not fully sure why, any ideas?

Reply:
Can you rebase onto the current master?
It is caused by the specified precedence (Ctrl+[) also matches and it is the first match.
But I added a change that should also sort them by the number of modifiers used, this would be a good test to see if my code is working.

Reply:
If you're referring to 5b39776, I already had it in my branch history. I updated to latest anyway and the issue wass still occurring. Even when manually reordering the entries in code, or removing all other actions it still wouldn't work.

I found however that I was able to get it working by adding another two keybinds, specifically for `{` & `}`, which are the shift-equivalents of `[` & `]` on most keyboards.

Reply:
Ah, that makes a lot of sense, glad you got it working. For these issues it is also helpful to pres `3` to get debug output in the console.
But this will still cause issues, on my keyboard that would be the `λ` and `χ` keys… that will potentially be resolved when we switch to hardware key presses. Should we switch to something else in the meantime?

Reply:
Ah that's handy to know about - I've just been littering the code with debug logs up until now 😄

Hm, I see. I'm not overly familiar with handling inputs in web apps so I'm not sure what an alternative solutions there are, besides perhaps trying to guess what kind of keyboard the user has or supporting key remapping.

Reply:
Yeah this kinda belongs to #124 and has yet to be addressed, it has already caused issues in the past